tests/mock_crust/utils.rs

```diff
@@ -40,18 +40,15 @@ const BALANCED_POLLING: bool = true;
 pub fn gen_range_except<T: Rng>(rng: &mut T,
                                 low: usize,
                                 high: usize,
-                                exclude: Option<usize>)
+                                exclude: BTreeSet<usize>)
```

Comment:
…or `exclude: I` with `I: IntoIterator<usize>`, then it would accept vectors and `None` and `Some` etc.